Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -702,14 +703,16 @@ def get_team_aad_id(team_name: str) -> str:
     :param team_name: Team name to get AAD ID of
     :return: team AAD ID
     """
+    demisto.debug(f'Team String {team_name}')
     integration_context: dict = get_integration_context()
     if integration_context.get('teams'):
         teams: list = json.loads(integration_context['teams'])
         for team in teams:
             if team_name == team.get('team_name', ''):
                 return team.get('team_aad_id', '')
-    url: str = f"{GRAPH_BASE_URL}/beta/groups?$filter=resourceProvisioningOptions/Any(x:x eq 'Team')"
+    url: str = f"{GRAPH_BASE_URL}/beta/groups?$filter=displayName eq '{team_name}' and resourceProvisioningOptions/Any(x:x eq 'Team')"
```

Comment:
shouldn't it be '&' rather than 'and'?

Reply:
The 'and' is correct per https://learn.microsoft.com/en-us/graph/filter-query-parameter.

Example:

List all users whose company name isn't undefined (that is, not a null value) or Microsoft.	GET ../users?$filter=companyName ne null and NOT(companyName eq 'Microsoft')&$count=true. This is an advanced query.